js/stringProcessing/relevantWords.js

```diff
@@ -199,17 +199,15 @@ function filterOnDensity( wordCombinations, wordCount, densityLowerLimit, densit
 }
 
 /**
- * Filters the list of word combination objects.
- * Word combinations with specific parts of speech at the beginning and/or end are removed.
+ * Filters the list of word combination objects based on the language-specific function word filters.
+ * Word combinations with specific parts of speech are removed.
  *
  * @param {Array} combinations The list of word combination objects.
  * @param {Function} functionWords The function containing the lists of function words.
  * @param {string} locale The paper's locale.
  * @returns {Array} The filtered list of word combination objects.
  */
-function filterCombinations( combinations, functionWords, locale ) {
-	combinations = filterFunctionWordsAnywhere( combinations, specialCharacters );
-	combinations = filterOneCharacterWordCombinations( combinations );
+function filterFunctionWords( combinations, functionWords, locale ) {
```

Comment:
The fix seems pretty minimal to me, since you are barely under the maximum allowed statements.
Looking at the code there are a lot of similar blocks of code like the following block:
```
combinations = filterFunctionWordsAnywhere( combinations, functionWords().transitionWords );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().adverbialGenitives );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().personalPronouns );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().reflexivePronouns );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().interjections );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().cardinalNumerals );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().verbs );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().indefinitePronouns );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().conjunctionsFilteredEverywhere );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().interrogatives );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().relativePronouns );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().locativeAdverbs );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().miscellaneous );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().prepositionalAdverbs );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().pronominalAdverbs );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().recipeWords );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().timeWords );
	combinations = filterFunctionWordsAnywhere( combinations, functionWords().vagueNouns );
```
Since this block calls the same function for different kinds of function words, could this not be extracted to a new function? I think this would really improve readability of the code and prevent the ESLint in the nearby future when new statements need to be added to this function.